### PR TITLE
Add toolbar and headline section

### PR DESCRIPTION
**Pull request by diwakarstha**
Create portfolio page
- add index.html file
- add style.css file
- add toolbar section in index.html
- add headline section in index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="wrapper">
+      <header class="nav">
+        <ul>
+          <li class="profile-name">Diwakar Shrestha</li>
+          <!-- <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li class="mail">
+            <img src="images/mail.png" alt="contact" />
+          </li> -->
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [x] Kindly remove commented codes as commented code makes our code base unreadable and not clean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="nav">
+        <ul>
+          <li class="profile-name">Diwakar Shrestha</li>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] This is supposed to be the logo according to project design, kindly use the `<a>` tag as it is being required in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md). making your logo a `<a>` tag is good for semantic reasons

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          <!-- <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li class="mail">
+            <img src="images/mail.png" alt="contact" />
+          </li> -->
+          <li><img src="images/toggle.png" alt="toggle-bar" /></li>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] According to [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), kindly wrap the toggle bar in `nav` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        </ul>
+      </header>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] It is a good practice for a frontend developer to follow the Figma design given. Your design is near perfection, I think it is better to have a space between the logo and toggle bar. kindly follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)
Figma design 👇 
                        
image

Your design 👇 
                  
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+      <section class="headline">
+        <p class="name"><strong>Diwakar Shrestha</strong></p>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] kindly change the `p` tag to `h1` tag as requested in the project requirement. `h1` as the main title is good for semantic and SEO reasons. kindly check this [HTML and CSS best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        <p class="position">Software Engineer</p>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] Also according to project requirement kindly change this subheading to `heading` tag i.e `h2` or `h3`. check this [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

File: style.css [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+/*
+  Header
+*/
```
Comment (VICTOR-CHUKWU) [inline style.css]: - [ ] Just as mentioned above, It would be nice if we keep our code base clean by removing commented codes

File: style.css [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+  flex: 1;
+}
+
+.wrapper .nav ul .mail {
+  flex: 1;
+}
+
+/*
+  Headline Section
+*/
```
Comment (VICTOR-CHUKWU) [inline style.css]: - [ ] Kindly remove comments as code, you did a nice job naming your class in a descriptive way, therefore is no need for the comments

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @diwakarstha  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit message
- Descriptive README file
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉 💯 👏🏻 

### Hi @diwakarstha 👋🏻 , 

Congratulations dear! You did a great job making changes requested by the previous reviewer 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**